dlp/api/Snippets.Tests/UpdateStoredInfoTypesTests.cs

```diff
@@ -28,45 +28,40 @@ public class UpdateStoredInfoTypesTests : IClassFixture<DlpTestFixture>
         [Fact]
         public void TestUpdate()
         {
-            Random random = new Random();
-            // Create the bucket for creating stored infoType.
-            var storage = StorageClient.Create();
-            var bucket = storage.CreateBucket(_fixture.ProjectId, $"dlp_update_dotnet_test_bucket_samples_{random.Next()}");
+            // Create an input bucket.
+            using var randomInputBucketFixture = new RandomBucketFixture();
+            var inputBucketName = randomInputBucketFixture.BucketName;
+            using var inputBucketCollector = new BucketCollector(inputBucketName);
 
-            // Create output bucket.
-            var bucketOutput = storage.CreateBucket(_fixture.ProjectId, $"dlp_update_dotnet_test_stored_infotype_bucket_{random.Next()}");
+            // Upload file to bucket.
+            var fileName = "test.txt";
+            var objectName = $"gs://{inputBucketName}/{fileName}";
+            inputBucketCollector.CopyToBucket(Path.Combine(_fixture.ResourcePath, "test.txt"), fileName);
 
-            // Upload the file to cloud storage.
-            var filePath = Path.Combine(_fixture.ResourcePath, "test.txt");
-            using var fileStream = File.OpenRead(filePath);
-            storage.UploadObject(bucket.Name, "test.txt", null, fileStream);
+            // Create an output bucket.
+            using var randomOutputBucketFixture = new RandomBucketFixture();
+            var outputBucketName = randomOutputBucketFixture.BucketName;
+            using var outputBucketCollector = new BucketCollector(outputBucketName);
 
-            try
-            {
-                // Create Stored InfoType first.
-                var storedInfoTypeId = $"github-usernames-update-{random.Next()}";
-                var response = CreateStoredInfoTypes.Create(_fixture.ProjectId, $"gs://{bucket.Name}", storedInfoTypeId);
+            Random random = new Random();
```

Comment:
```suggestion
```

Reply:
We are appending some random number after the name of Stored InfoType. We are going to need this.

Reply:
Oh sorry, didn't see that. Thought it was just leftover from bucket creation.